src/core/ddsi/src/q_misc.c

```diff
@@ -42,6 +42,16 @@ int WildcardOverlap(char * p1, char * p2)
 }
 #endif
 
+int guid_prefix_eq (const ddsi_guid_prefix_t *a, const ddsi_guid_prefix_t *b)
+{
+  return a->u[0] == b->u[0] && a->u[1] == b->u[1] && a->u[2] == b->u[2];
+}
+
+int guid_eq (const struct ddsi_guid *a, const struct ddsi_guid *b)
```

Comment:
There is a ``entity_guid_eq`` in ``q_entity.c``. That one should be removed.

(And I dislike the superfluous parentheses.)

Reply:
I did not the entity_guid_eq because it references entities and guid_eq does not.

Reply:
Good point — don't worry about it for now